lib/launcher.js

```diff
@@ -351,11 +351,16 @@ class Launcher {
 
     /**
      * emit event from child process to reporter
-     * @param  {Object} m  event object
+     * @param  {String} cid
+     * @param  {Object} m event object
      */
-    messageHandler (m) {
+    messageHandler (cid, m) {
```

Comment:
Yes. This is correct, because [here](https://github.com/webdriverio/webdriverio/blob/master/lib/launcher.js#L326) we pass variable `rid` as variable `cid`. And [here](https://github.com/webdriverio/webdriverio/blob/master/lib/runner.js#L441) we subscribe on messages and [here](https://github.com/webdriverio/webdriverio/blob/master/lib/runner.js#L21) we save this `cid` (which is actually `rid`).